pkg/multiTenancyPlugins/executor.go

```diff
@@ -28,10 +28,11 @@ func (*Executor) Handle(cluster cluster.Cluster, swarmHandler http.Handler) http
 	}
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		log.Debug(r)
-		err := startHandler(utils.ParseCommand(r), cluster, w, r, swarmHandler)
-		if err != nil {
-			log.Error(err)
-			http.Error(w, err.Error(), http.StatusBadRequest)
+		var errInfo utils.ErrorInfo
```

Comment:
maybe put here errInfo.Status = http.StatusBadRequest  as default and remove it from all callers?  Or maybe set it as default in the struct itself